Makefile

```diff
@@ -28,15 +28,17 @@ install-nltk-data:
 
 
 STATIC_JS = openassessment/xblock/static/js
+STATIC_CSS = openassessment/xblock/static/css
 
-javascript:
-	node_modules/.bin/uglifyjs $(STATIC_JS)/src/oa_shared.js $(STATIC_JS)/src/*.js $(STATIC_JS)/src/lms/*.js > "$(STATIC_JS)/openassessment-lms.min.js"
-	node_modules/.bin/uglifyjs $(STATIC_JS)/src/oa_shared.js $(STATIC_JS)/src/*.js $(STATIC_JS)/src/studio/*.js > "$(STATIC_JS)/openassessment-studio.min.js"
-
+javascript: update-npm-requirements
+	node_modules/.bin/uglifyjs $(STATIC_JS)/src/oa_shared.js $(STATIC_JS)/src/*.js $(STATIC_JS)/src/lms/*.js $(STATIC_JS)/lib/backgrid/backgrid.min.js -c > "$(STATIC_JS)/openassessment-lms.min.js"
+	node_modules/.bin/uglifyjs $(STATIC_JS)/src/oa_shared.js $(STATIC_JS)/src/*.js $(STATIC_JS)/src/studio/*.js $(STATIC_JS)/lib/backgrid/backgrid.min.js -c > "$(STATIC_JS)/openassessment-studio.min.js"
 
 sass:
 	python scripts/compile_sass.py
 
+verify-generated-files: javascript sass
+	git diff --exit-code
```

Comment:
@andy-armstrong @bjacobel I've added this stanza to Travis builds in order to prevent merging javascript/sass updates without updating the bundles we store on git. Good idea/bad idea/better suggestion?